cmd/builder/builder_options_test.go

```diff
@@ -36,7 +37,12 @@ func TestApplyOptions(t *testing.T) {
 		Options.LoggerOption(zap.NewNop()),
 		Options.MetricsFactoryOption(metrics.NullFactory),
 		Options.MemoryStoreOption(memory.NewStore()),
+		Options.ElasticSearchOption(&escfg.Configuration{
+			Servers: []string{"127.0.0.1"},
```

Comment:
I think we should fail fast and violently when multiple storage options are specified, for e.g. `MemoryStoreOption` and `ElasticSearchOption`. The current behavior of silently configuring an arbitrary store when multiple storage options could lead to lot of wasted time debugging.